src/components/Referrals/Referrals.container.spec.ts

```diff
@@ -0,0 +1,49 @@
+import { RootState } from '../../modules/reducer'
+import { fetchReferralsRequest } from '../../modules/referrals/actions'
+import { getError, isLoadingReferrals, getInvitedUsersAccepted, getInvitedUsersAcceptedViewed } from '../../modules/referrals/selectors'
+
+describe('Referrals Container', () => {
```

Comment:
There's no need to test containers, we don't usually test them, you are free to remove them!